Doc/c-api/sys.rst

```diff
@@ -258,10 +258,55 @@ These are utility functions that make functionality from the :mod:`sys` module
 accessible to C code.  They all work with the current interpreter thread's
 :mod:`sys` module's dict, which is contained in the internal thread state structure.
 
+.. c:function:: PyObject *PySys_GetAttr(PyObject *name)
+
+   Get the attribute *name* of the :mod:`sys` module. Return a :term:`strong reference`.
+   Raise :exc:`RuntimeError` and return ``NULL`` if it does not exist.
```

Comment:
I would prefer to raise AttributeError.

Reply:
Then we would need a new pair of functions that raise RuntimeError to use in the CPython code. Or you can add a pair of functions that raise AttributeError later.

Reply:
> Then we would need a new pair of functions that raise RuntimeError to use in the CPython code.

Would you mind to elaborate why RuntimeError is needed by CPython?

Reply:
Status quo. And it looks to me it is justified by the purpose of RuntimeError: "Raised when an error is detected that doesn’t fall in any of the other categories."

If you want to change RuntimeError's in the existing code to something other, you should justify that change.

Reply:
In my implementation https://github.com/python/cpython/pull/129369, `PySys_SetAttr()` raises `RuntimeError` if the `sys` module cannot be retrieved. But it raises `AttributeError` if the attribute doesn't exist.

`PySys_GetObject()` ignores all errors and returns `NULL` on error or if the attribute doesn't exist. So I'm confused, which function using `PySys_GetObject()` currently rely on the `RuntimeError`? Do you have an example?

Reply:
About a half of functions that use `PySys_GetObject()` or `_PySys_GetAttr()` raise RuntimeError if they return NULL. See my statistics in https://github.com/capi-workgroup/decisions/issues/54. Look at usages of `PySys_GetAttr()` and `PySys_GetAttrString()` in this PR.

Reply:
The C API Working Group voted for RuntimeError.